AberrantSMPP.Tests/AberrantSMPP.Tests.csproj

```diff
@@ -11,8 +11,9 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>AberrantSMPP.Tests</RootNamespace>
     <AssemblyName>AberrantSMPP.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Try with net45, as this is currently our lower barrier form libraries.

Reply:
Okey, in the actual commit, the TargetFramework is set to 4.5